### PR TITLE
OpenConfig support

### PR DESCRIPTION
**Pull request by martin-volf**
Add a couple of OC modules to ConfD; this requires some fixes in how paths are handled.
Note that it depends on #103.

### CODE REVIEW

File: confdgnmi/oc-comps.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<config xmlns="http://tail-f.com/ns/config/1.0">
```
Comment (jomiklos) [inline confdgnmi/oc-comps.xml]: `oc-comps.xml` sounds a bit crptc, maybe rename to `oc-components.xml`?

File: confdgnmi/oc-if.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<config xmlns="http://tail-f.com/ns/config/1.0">
```
Comment (jomiklos) [inline confdgnmi/oc-if.xml]: `oc-if.xml` -> `oc-interfaces.xml` for more clarification?

File: confdgnmi/src/confd_gnmi_api_adapter.py [hunk 3]
```diff
@@ -497,11 +497,12 @@ def make_gnmi_json_value(self, value, csnode):
 
     def get_updates(self, trans, path_str, save_flags):
         log.debug("==> path_str=%s", path_str)
-        csnode = _confd.cs_node_cd(None, path_str)
+        tagpath = '/' + '/'.join(tag for tag, keys in parse_instance_path(path_str))
```
Comment (jomiklos) [inline confdgnmi/src/confd_gnmi_api_adapter.py]: `keys` are unused, use `_` instead as throwaway?

File: confdgnmi/src/confd_gnmi_common.py [hunk 4]
```diff
@@ -83,24 +83,31 @@ def make_name_keys(elem_string) -> Tuple[str, Dict[str, str]]:
...
 
 
-# Crate gNMI Path object from string representation of path
-# see: https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#222-paths
-# TODO tests
-def make_gnmi_path(xpath_string: str, origin: str = None, target: str = None) -> gnmi_pb2.Path:
-    """ Create gNMI path from input string. """
-    log.debug("==> xpath_string=%s origin=%s target=%s", xpath_string, origin, target)
+def parse_instance_path(xpath_string) -> Iterable[Tuple[str, List[Tuple[str, str]]]]:
+    """Parse instance path to a iterable of (tag, keys) tuples.
```
Comment (jomiklos) [inline confdgnmi/src/confd_gnmi_common.py]: "an" iterable

**Review by jomiklos — COMMENTED** [review]
cosmetic/ignorable comments...

**Review by micnovak — APPROVED** [review]

**Review by jomiklos — APPROVED** [review]

**Pull request merged**